lib/manageiq/api/client/collection.rb

```diff
@@ -42,6 +43,58 @@ def self.subclass(name)
             const_set(klass_name, klass)
           end
         end
+
+        def ar_search(options)
+          search(query_parameters_from_ar_query_options(options))
+        end
+
+        private
+
+        def query_parameters_from_ar_query_options(options)
+          api_params = {}
+          [:offset, :limit].each { |opt| api_params[opt] = options[opt] if options[opt] }
+          api_params[:attributes] = options[:select].join(",") if options[:select].present?
+          if options[:where]
+            api_params[:filter] ||= []
+            conditional_parameter_from_ar_query_options(api_params[:filter], "=", options[:where])
+          end
+          if options[:not]
+            api_params[:filter] ||= []
+            conditional_parameter_from_ar_query_options(api_params[:filter], "!=", options[:not])
+          end
+          if options[:order]
+            order_parameters_from_ar_query_options(api_params, options[:order])
+          end
+          api_params
+        end
+
+        def conditional_parameter_from_ar_query_options(filter_param, condition, ar_option)
+          ar_option.each do |attr, values|
+            Array(values).each do |value|
+              value = "'#{value}'" if value.kind_of?(String) && !value.match(/^(NULL|nil)$/i)
```

Comment:
This feel like it will have escaping issues.  That is if the value has a `'` in it, what will happen.